R/tag_query.R

```diff
@@ -1088,40 +1093,40 @@ tagQueryAttrsRemove <- function(els, attrs) {
   }
   tagQueryWalk(els, function(el) {
     if (!isTagEnv(el)) return()
-    for (attrVal in attrs) {
-      el$attribs[[attrVal]] <- NULL
-    }
-  })
-}
-# Remove attribute values
-tagQueryAttrsEmpty <- function(els) {
-  tagQueryWalk(els, function(el) {
-    if (!isTagEnv(el)) return()
-    el$attribs <- list()
+    tagEnvRemoveAttribs(el, attrs)
   })
 }
 # Check if els have attributes
-tagQueryAttrHas <- function(els, attr) {
-  attr <- as_character2(attr)[[1]]
-  if (length(attr) != 1 || !is.character(attr)) {
-    stop("`attr` must be a single character value")
+tagQueryAttrsHas <- function(els, attrs) {
+  attrs <- as_character2(attrs)
+  if ((length(attrs) == 0) || (!is.character(attrs))) {
+    stop("`attrs` must be a character vector")
```

Comment:
Maybe I missed something but I had thought all of the `class` methods we're going to allow a character vector? That seems sensible to me, but if this is the behavior you want I'll have to roll back some of the doc changes I made in

Reply:
I'm confused. This is in the `$hasAttrs(attrs)` method.

The class methods still allow vectors.

Reply:
Ahh, my bad, I meant to add that comment here https://github.com/rstudio/htmltools/pull/262/files#diff-3299ef0635ccf99ae3a3ea996dcd596257444b4a034e704d408a921163f42a02R1152

Reply:
Good catch!